nwbinspector/checks/nwbfile_metadata.py

```diff
@@ -38,12 +40,35 @@ def check_session_start_time_future_date(nwbfile: NWBFile):
 
 
 @register_check(importance=Importance.BEST_PRACTICE_SUGGESTION, neurodata_type=NWBFile)
-def check_experimenter(nwbfile: NWBFile):
+def check_experimenter_exists(nwbfile: NWBFile):
     """Check if an experimenter has been added for the session."""
     if not nwbfile.experimenter:
         return InspectorMessage(message="Experimenter is missing.")
 
 
+@register_check(importance=Importance.BEST_PRACTICE_SUGGESTION, neurodata_type=NWBFile)
+def check_experimenter_form(nwbfile: NWBFile):
+    """Check the text form of each experimenter to see if it matches the DANDI regex pattern."""
+    if not is_module_installed(module_name="dandi"):
+        warn(
+            "It is strongly recommended to download DANDI alongside the NWB Inspector \n\n"
+            "\tpip install dandi\n\nor\n\n\tpip install nwbinspector.[dandi]\n"
+        )
+        return
+
+    from dandischema.models import NAME_PATTERN
+
+    for experimenter in nwbfile.experimenter:
+        experimenter = experimenter.decode() if isinstance(experimenter, bytes) else experimenter
+        if re.match(string=experimenter, pattern=NAME_PATTERN) is None:
+            yield InspectorMessage(
+                message=(
+                    f"The name of experimenter '{experimenter}' does not match the DANDI form "
+                    "(Last, First Middle or Last, First M.)."
+                )
+            )
```

Comment:
Maybe if dandi isn't installed we can default to implementing this pattern locally

Reply:
Sure thing, added.